assets/src/components/account/AccountOverview.tsx

```diff
@@ -29,6 +30,17 @@ type State = {
   isLoading: boolean;
   isEditing: boolean;
   isRefreshing: boolean;
+  hideWidgetOutsideWorkingHours: boolean;
+};
+type Account = {
```

Comment:
just a thought, could also be AccountShow if we're interested in telling the difference between Account#basic.json and Account#account.json (could def also be just `?` optional keys)

Reply:
aha! sounds good 👍